CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+
+## v1.0.5
+### 2024-08-19
+
+This version of HOSS updates the 'is_index_subset' method to check for empty list (in case of dimensions subset) 
```

Comment:
Super pedantic nit: The single-quotes around the `is_index_subset` should probably be backticks so that the function renders as code in the markdown.